openmetadata-ui/src/main/resources/ui/src/components/common/DomainLabel/DomainLabel.component.tsx

```diff
@@ -87,7 +88,7 @@ export const DomainLabel = ({
         <Link
           className={classNames(
             'text-primary no-underline domain-link',
-            !showDomainHeading ? 'font-medium text-xs' : ''
+            !showDomainHeading ? `font-medium text-xs ${className}` : ''
           )}
```

Comment:
We are only applying the className based on condition `!showDomainHeading`, why is that? shouldn't it be applied without any conditions?

Reply:
No, so we don't this CSS when the `showDomainHeading` is true, to match the label and value styling.